webknossos/webknossos/dataset/dataset.py

```diff
@@ -593,11 +593,21 @@ def _disambiguate_remote(
         dataset_name: str,
         organization_id: str,
     ) -> str:
+        from ..client.context import _get_context
+
+        current_context = _get_context()
         possible_ids = list(
             cls.get_remote_datasets(
                 name=dataset_name, organization_id=organization_id
             ).keys()
         )
+        if (
```

Comment:
Would this make sense?
```suggestion
        if len(possible_ids) == 0 and (
```

Reply:
Yes, I changed it.